crypto/conf/conf_def.c

```diff
@@ -9,29 +9,40 @@
 
 /* Part of the code in here was originally in conf.c, which is now removed */
 
+#include "e_os.h"
```

Comment:
As a rule of thumb less e_os.h is include better. So what is it used for here? If I removed it it still compiles...

Reply:
I see S_IFDIR to be defined in e_os.h on some systems. Maybe it is not needed anymore?

Reply:
I'd still be in favour of removing e_os.h. The condition for S_IFDIR [re]definition is kind of DJGPP or WIN32, but DJGPP has S_IFDIR, and there is S_IFDIR even on Windows by default. As far as I can tell It "disappears" if you "disable [language] extensions", but it might as well be problematic for a lot of other reasons...

Reply:
For reference, attempt to disable [language] extensions" results in errors in Windows system headers, so it's not really an option. In other words omit e_os.h.